exe/Parser/Text.hs

```diff
@@ -547,27 +542,31 @@ type ModuleRecipe a = (String, String -> Fold IO a String)
 -- GeneratorRecipe is a list of ModuleRecipe
 type GeneratorRecipe a = [ModuleRecipe a]
 
-fileEmitter :: String -> ModuleRecipe a -> Fold IO a ()
-fileEmitter outdir (modName, fldGen) = Fold.mapM action $ fldGen modName
+fileEmitter :: String -> String -> ModuleRecipe a -> Fold IO a ()
+fileEmitter file outdir (modName, fldGen) = Fold.mapM action $ fldGen modName
 
     where
 
+    pretext = "-- autogenerated from ucd/" ++ file ++ "\n"
```

Comment:
Would be nice if we can prepend the specific path with version number e.g.: `https://www.unicode.org/Public/13.0.0/ucd/UnicodeData.txt`.

Reply:
There are multiple ways of doing this,
- We can hardcode the path. But, on every version update, we'll have to change the path here manually.
- We can set an env variable with the version while running `run-ucd2haskell`. Ideally, we can combine both the script files, `download-ucd` and `run-ucd2haskell` so that the version needs to be updated only in 1 place.